compiler/testData/codegen/box/diagnostics/functions/tailRecursion/defaultArgsWithSideEffects3.kt

```diff
@@ -0,0 +1,33 @@
+// DONT_RUN_GENERATED_CODE: JS
```

Comment:
Seems this and next one tests are redundant

Reply:
Done.